indico/modules/users/models/users.py

```diff
@@ -574,3 +576,33 @@ def _user_deleted(target, value, *unused):
 define_unaccented_lowercase_index(User.last_name)
 define_unaccented_lowercase_index(User.phone)
 define_unaccented_lowercase_index(User.address)
+
+
+class RegistrationRequest(db.Model):
+    __tablename__ = 'registration_requests'
+    __table_args__ = (
+        db.Index(None, 'email', unique=True),
```

Comment:
you can add `unique=True` to the column definition instead. Also, add a CHECK constraint for `email = lower(email)` to enforce lowercase emails (and convert to lowercase if needed during registration, but i guess we already do that)